pkg/plugins/policies/meshretry/api/v1alpha1/zz_generated.resource.go

```diff
@@ -19,14 +24,32 @@ import (
 var rawSchema []byte
 
 func init() {
-	var schema spec.Schema
+	var structuralSchema *schema.Structural
+	var v1JsonSchemaProps *apiextensionsv1.JSONSchemaProps
+	var validator *validate.SchemaValidator
 	if rawSchema != nil {
-		if err := yaml.Unmarshal(rawSchema, &schema); err != nil {
+		rawJson, err := yaml.YAMLToJSON(rawSchema)
+		if err != nil {
 			panic(err)
 		}
+		if err := json.Unmarshal(rawJson, &v1JsonSchemaProps); err != nil {
```

Comment:
you can just call `yaml.Unmarshal(rawSchema, &v1JsonSchemaProps)` to avoid calling `YAMLToJSON` function

Reply:
I can't because it won't unmarshal properly because there are no custom functions defined on yaml https://github.com/kumahq/kuma/pull/12829#discussion_r1952185780

Reply:
I see, that's actually how `yaml.Unmarshal` works, it calls `yaml.YAMLToJSON` inside https://github.com/kubernetes-sigs/yaml/blob/master/yaml.go#L43. So doesn't really matter